CHANGELOG.md

```diff
@@ -18,7 +18,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 
 ### Removed
 
-## [v0.34.0](https://github.com/thanos-io/thanos/tree/release-0.34) - release in progress
+## [v0.34.0](https://github.com/thanos-io/thanos/tree/release-0.34) - 26-01-24
```

Comment:
Can we follow the same date format? https://github.com/thanos-io/thanos/pull/7095/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR51

Reply:
oh yeah i spaced out there, thanks!

Reply:
fixed!